setup.py

```diff
@@ -24,7 +24,7 @@ def read(*parts):
     url='https://github.com/python-babel/django-babel/',
     packages=find_packages(exclude=('tests',)),
     install_requires=[
-        'django>=1.4,<1.10',
+        'django>=1.4,<1.11',
         'babel>=1.3',
     ],
     classifiers=[
```

Comment:
you can put the settings file in `tests/dj_babel_test_utils/example_project/settings.py`

It will then be available on the `PYTHONPATH` at `dj_babel_test_utils.example_project.settings`

you'll need to update the other files too.